src/global/ApolloLinkState/persona.ts

```diff
@@ -0,0 +1,86 @@
+import { Resolvers } from 'apollo-boost';
+
+import { GET_CARD, GET_PAGE, GET_PERSONA_STEP_PATH } from 'global/graphqls/Persona';
+
+import { Persona, PersonaCard, PersonaPage } from './namespace';
+
+const cardDefaults: PersonaCard = {
+  __typename: 'PersonaCard',
+  name: '',
+  description: '',
+  avatar: '',
+  background: '',
+};
+
+const pageDefaults: PersonaPage = {
+  __typename: 'PersonaPage',
+  background: '',
+  avatar: '',
+  content: '',
+};
+
+const personaDefaults: Persona = {
+  __typename: 'Persona',
+  personaStepPath: '',
+  card: cardDefaults,
+  page: pageDefaults,
+  personaUUIDs: [],
+};
+
+const personaResolvers: Resolvers = {
+  Mutation: {
+    changePersonaStepPath: (_, { personaStepPath }: Persona, { cache }) => {
```

Comment:
So you have chosen to manage stepper with apollo state instead of url address? What about back buttons on mobile, will they be supported too?

Reply:
this is reworked, please once more time